docs/src/modules/components/AppLayoutDocsFooter.js

```diff
@@ -110,16 +110,15 @@ async function postFeedbackOnSlack(data) {
   }
 
   try {
-    // await fetch(`${window.location.origin}/.netlify/functions/feedback-management/`, {
-    await fetch(
-      `https://deploy-preview-36472--material-ui.netlify.app//.netlify/functions/feedback-management/`,
-      {
-        method: 'POST',
-        headers: { 'content-type': 'application/x-www-form-urlencoded' },
-        // Seems tricky but it's to match how slack send data
-        body: `payload=${encodeURIComponent(JSON.stringify(sentData))}`,
-      },
-    );
+    const res = await fetch(`${window.location.origin}/.netlify/functions/feedback-management/`, {
+      method: 'POST',
+      headers: { 'content-type': 'application/x-www-form-urlencoded' },
+      // Seems tricky but it's to match how slack send data
```

Comment:
Seems the right way to me, one way to make this a bit cleaner could be by using `URLSearchParams` but essentially this boils down to the exact same thing.

```tsx
body: new URLSearchParams({
  payload: JSON.stringify(sentData)
})
```